src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  top: 10px;
+  bottom: 10px;
+  height: 100%;
+  width: 100%;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+  justify-content: flex-end;
+  /* background-color: blue; */
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  /* background-color: red; */
+}
+
+.nav__item {
+  margin-left: 20px;
+  text-transform: uppercase;
+  /* background-color: orange; */
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  text-decoration: none;
+  color: black;
+  height: 60px;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
transition of color should be smooth whe you hover on the text so add `transition` property for the here

Reply:
cool.